VideoAPI/VideoApi.Domain/Enums/EventType.cs

```diff
@@ -12,6 +12,7 @@ public enum EventType
         Leave,
         Consultation,
         JudgeAvailable,
-        MediaPermissionDenied
+        MediaPermissionDenied,
+        Joining
```

Comment:
Change to 'ParticipantJoining' to be more verbose about the event type

Reply:
updated the eventType name.